src/YarpUtilities/include/BipedalLocomotionControllers/YarpUtilities/Helper.h

```diff
@@ -20,6 +20,8 @@
 
 #include <iDynTree/Core/VectorDynSize.h>
 
+#include <BipedalLocomotionControllers/TemplateHelpers.h>
```

Comment:
Why not `#include <BipedalLocomotionControllers/GenericContainer/TemplateHelpers.h>`?

According to `add_bipedal_component` `cmake` function the files are installed in 
https://github.com/dic-iit/bipedal-locomotion-controllers/blob/e055599aa3cbe6bc3f4976a53c358694a5c0970a/cmake/AddBipedalLocomotionControllersComponent.cmake#L87

Reply:
I used only the version in the build folder actually. I did not notice. Is it ok for you if we include there a file of another component?

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-controllers/pull/29/commits/2fd26da1dcaac5814f255c8e7d2ba2cea0dcae15